### PR TITLE
CAMEL-14674: Nicer table design for the blog post

### PR DESCRIPTION
**Pull request by aashnajena**
Styled tables appearing in Blogs for Apache Camel Website
In response to Issue 14674 : https://issues.apache.org/jira/browse/CAMEL-14674

- added bottom margin for table
- added odd-even distinction for table rows
- added padding for table cells
- added different colour and more padding to table headers
- centre-aligned the table and text in the table
- widened the table to align with the content written around it

css_table_update

### CODE REVIEW

File: antora-ui-camel/src/css/blog.css [hunk 1]
```diff
@@ -99,6 +99,25 @@ article.blog p {
...
 }
 
+.blog .post .post-content table {
+  width: 100%;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.blog .post .post-content tr:nth-child(even) {
+  background: #f4f5f7;
```
Comment (zregvart) [inline antora-ui-camel/src/css/blog.css]: Can we use one of the variables in `vars.css` instead? `--color-smoke-50` looks similar.

File: antora-ui-camel/src/css/blog.css [hunk 2]
```diff
@@ -99,6 +99,25 @@ article.blog p {
...
+}
+
+.blog .post .post-content td {
+  padding: 0.5rem;
+}
+
+.blog .post .post-content th {
+  padding: 0.7rem;
+  background: #e1e4e9;
```
Comment (zregvart) [inline antora-ui-camel/src/css/blog.css]: Let's rather use the CSS variables here?

**Comment by zregvart** [comment]
Thanks @aashnajena, note that you can edit the title of the pull request so when reviewing folk know what the pull request is about. In this case a good title would be `CAMEL-14674: Nicer table design for the blog post`

File: antora-ui-camel/src/css/blog.css (hunk 1 above)
Comment (aashnajena) [reply antora-ui-camel/src/css/blog.css]: I have updated the colours from vars.css. Please take a look now!

**Comment by zregvart** [comment]
@aashnajena I noticed you also added changes from another issue in c8973318bad6cf8944293b4aae161927cedc488e if it's not too difficult for you can you move that to a new pull request? The reason is that we can review and merge different issues separately, making it all or nothing by bundling multiple issues in the same pull request prolongs the process and makes it difficult to review.

**Comment by aashnajena** [comment]
> Perhaps split into two pull requests

Hi, I have made those changes in a separate branch. Is it affecting this PR as well?

**Comment by zregvart** [comment]
> Hi, I have made those changes in a separate branch. Is it affecting this PR as well?

Yeah, you can see the commits included in this pull request on the Commits page:
https://github.com/apache/camel-website/pull/181/commits

**Comment by aashnajena** [comment]
Hey! I'm so sorry, the new branch I had made was branching out from this one, instead of the master. I have made a new branch and done the commits related to the other issue. I have opened a new pull request for that. 

What should I do about this PR? Should I revert back the changes and commit just the blog.css file again, or is it better if I close this PR and raise a new one (since changes are small, I can redo them easily)

**Comment by zregvart** [comment]
@aashnajena can I ask that you rebase this branch?

**Comment by aashnajena** [comment]
Hi, I have rebased this branch. Now you should only be able to see commits related to [this](https://issues.apache.org/jira/browse/CAMEL-14674) issue. Thanks for helping me!

**Comment by zregvart** [comment]
@aashnajena perfect, thanks, let's wait for the preview to make sure everything looks good and then we can merge this.

**Comment by zregvart** [comment]
@aashnajena I just noticed that the previous non sectioned rules remain in the blog.css, can you also remove these lines?

https://github.com/apache/camel-website/pull/181/files#diff-84ac33c1784edcb2c266b9fdc2ca36bbR156-R178

**Comment by aashnajena** [comment]
Hey! I wasn't sure if I should remove those, since they weren't my commits. I have removed them now!

**Comment by zregvart** [comment]
Thanks @aashnajena!

**Pull request merged**